econet_hvac_odu.yaml

```diff
@@ -244,3 +251,31 @@ sensor:
     device_class: energy
     unit_of_measurement: "kWh"
     accuracy_decimals: 1
+
+text_sensor:
+  - platform: econet
+    name: "ODU Model Number"
+    sensor_datapoint: PRODMODN
+    id: odu_model_no
+    request_once: true
+    request_mod: 7
+    src_address: ${odu_address}
+    icon: "mdi:information-box"
+    entity_category: "diagnostic"
+  - platform: econet
+    name: "ODU Serial Number"
+    sensor_datapoint: PRODSERN
+    id: odu_Serial_no
+    request_once: true
+    request_mod: 7
+    src_address: ${odu_address}
+    icon: "mdi:information-box"
+    entity_category: "diagnostic"
+  - platform: econet
+    name: "ODU Software Version"
+    id: odu_sw_version
+    sensor_datapoint: ODUSWVER
+    request_once: true
+    src_address: ${thermostat_address}
```

Comment:
Is this correct? This datapoint comes from the Thermostat rather than the ODU?

Reply:
Yes.

  I initially tried just requesting from the ODU. [1024 decimal, 0x400 hex]. But either our implementation has something funky, or there is a bug in the Rheem econet implementation - when I would request from just the ODU, the thermostat AND the furnace would also reply, and which order they replied in would determine the value displayed.

  So, given that this datapoint is available, I dropped it in as it got the job done.